File: igor/Problema0103/problema0103.java
import java.util.Scanner;
public class problema0103 {

    static Scanner input = new Scanner(System.in);
    public static void main(String[] args) {
        System.out.println("escreve o numero do dia atual: ");
        int diaAtual = input.nextInt();
        System.out.println("escreve o numero do mes atual: ");
        int mesAtual = input.nextInt();
        System.out.println("escreve o numero do ano atual: ");
        int anoAtual = input.nextInt();


        System.out.println("escreve o numero do dia: ");
        int dia = input.nextInt();
        System.out.println("escreve o numero do mes: ");
        int mes = input.nextInt();
        System.out.println("escreve o numero do ano: ");
        int ano = input.nextInt();

        if(anoAtual < ano){
            System.out.println("Nao esta vencida");
        }else if(anoAtual > ano){
            System.out.println("Esta vencida");
        }else{
            if(mesAtual < mes){
                System.out.println("Nao esta vencida");
            }else if(mesAtual > mes){
                System.out.println("Esta vencida");
            }else{
                if(diaAtual <= dia){
                    System.out.println("Nao esta vencida");
                }else {
                    System.out.println("Esta vencida");
                }
            }
        }

    }
}

File: igor/Problema0103/problema0103.py
diaAtual = int(input("Escreve o numero do dia atual: "))
mesAtual = int(input("Escreve o numero do mes atual: "))
anoAtual = int(input("Escreve o numero do ano atual: "))

dia = int(input("Escreve o numero do dia: "))
mes = int(input("Escreve o numero do mes: "))
ano = int(input("Escreve o numero do ano: "))

if anoAtual < ano:
    print("Nao esta vencida")
elif anoAtual > ano:
    print("Esta vencida")
else:
    if mesAtual < mes:
        print("Nao esta vencida")
    elif mesAtual > mes:
        print("Esta vencida")
    else:
        if diaAtual <= dia:
            print("Nao esta vencida")
        elif diaAtual > dia:
            print("Esta vencida")
